helm/charts/determined/templates/master-config.yaml

```diff
@@ -95,3 +95,41 @@ data:
     {{- if .Values.clusterName }}
     cluster_name: {{ .Values.clusterName }}
     {{- end }}
+
+    logging:
+      {{- if .Values.logging.backend }}
+      backend: {{ .Values.logging.backend }}
+      {{- end }}
+      {{- if .Values.logging.host }}
+      host: {{ .Values.logging.host }}
```

Comment:
question: should we put in checks to make sure these values are specified (make them required)?

Reply:
are these values used only if elastic is the backend? then we can just put all of this inside the if statement `{{ if .Values.logging.backend == elastic }}`

Reply:
ah yeah, ok. yeah, they are, we probably should.

Reply:
If we don't expect default values to work well for these fields (and I assume they don't) I think its a much better user experience if errors are surfaced when doing `helm install` vs having to chase down pod logs.

Reply:
Good ideas all around, done.